src/modules/media/components/ObjectPlaceholder/ObjectPlaceholder.types.ts

```diff
@@ -1,9 +1,11 @@
+import { ReactNode } from 'react';
+
 import { DefaultComponentProps } from '@shared/types';
 
 export interface ObjectPlaceholderProps extends DefaultComponentProps {
-	description?: string;
+	description?: string | ReactNode;
```

Comment:
`ReactNode` includes `string` already
```suggestion
	description?: ReactNode;
```